src/androidTest/java/androidx/content/res/TypedArrayTest.kt

```diff
@@ -204,4 +204,23 @@ class TypedArrayTest {
             array.getTextOrThrow(R.styleable.TypedArrayTypes_text_array_absent)
         }.hasMessageThat().isEqualTo("Attribute not defined in set.")
     }
+
+    @Test fun use() {
+        val attrs = context.getAttributeSet(R.layout.typed_array)
+        val array = context.obtainStyledAttributes(attrs, R.styleable.TypedArrayTypes)
+
+        var booleanPresent = false
+        array.use {
+            booleanPresent = it.getBooleanOrThrow(R.styleable.TypedArrayTypes_boolean_present)
+        }
+        assertTrue(
+            "Boolean obtained from attrs wasn't correct. " +
+                "Either the 'use' block wasn't run, or the boolean was missing from the TypedArray.",
```

Comment:
Since you call getBooleanOrThrow, isn't the only way to reach here that the block wasn't run?

Reply:
Yep, my mistake! Will update.